x-pack/plugins/canvas/public/components/workpad_page/workpad_interactive_page/index.js

```diff
@@ -111,13 +106,13 @@ const mapStateToProps = (state, ownProps) => {
 
 const mapDispatchToProps = dispatch => ({
   dispatch,
-  insertNodes: pageId => selectedNodes => dispatch(insertNodes(selectedNodes, pageId)),
-  removeNodes: pageId => nodeIds => dispatch(removeElements(nodeIds, pageId)),
+  insertNodes: (pageId, selectedNodes) => dispatch(insertNodes(selectedNodes, pageId)),
```

Comment:
nit: would be nice if this and the action had the same argument order, but it's also not a big deal that they don't.

Reply:
I went ahead and changed the arg order for `insertNodes` and `removeNodes` to match the redux action.